cmd/printer/interactive.go

```diff
@@ -104,6 +104,16 @@ func (p *InteractivePrinter) Bold(name string) *InteractivePrinter {
 	return p
 }
 
+func (p *InteractivePrinter) DangerBold(name string) *InteractivePrinter {
+	p.printOut(termenv.String(name).Bold().Foreground(p.profile.Color("1")).String())
```

Comment:
Non blocking: How easy would it be to turn these magic numbers into constants or something more readable?